modules/ROOT/pages/idempotent-message-validator.adoc

```diff
@@ -4,41 +4,69 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, filters, conditional, gates, idempotent
 
-Ensures that only unique messages continue through a flow's execution by checking the unique ID of the incoming message.
-
-== Idempotent Message Validator
-
-The Idempotent Message Validator ensures that only unique messages are passed on. It does this by checking the unique ID of the incoming message. To compute the unique ID, you can use a DataWeave expression, including the DataWeave `Crypto` functions that can compute hashes(SHA, MD5) from the data.
-
-=== Parameters
-
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
-|===
-| Name | Type | Description | Default Value | Required
-| ID Expression | Expression | The expression to use when extracting the ID from the message. | #[correlationId] | {nbsp}
-| Value Expression | Expression | The expression to use when extracting the value from the message. The result is what will be stored in the object store in case the Id is unique. | #[correlationId] | {nbsp}
-| Object Store | Object Store | Either a name to reference a global object store or a definition of a private object store where the processed message IDs will be stored. | Object Store created by the Object Store Manager. Not persistent, with an entry TTL of 5 MINUTES and an expiration interval of 6 SECONDS. | {nbsp}
-| Store Prefix | String | Defines the prefix of the object store names. This will only be used for the internally built object store. | _configFileName_._flowName_.IdempotentMessageValidator | {nbsp}
-|===
-
-==== Throws
-
-* `MULE:DUPLICATE_MESSAGE`
+The Idempotent Message Validator ensures that only unique messages continue through a flow's execution by checking the unique ID of the incoming message. You can use any incoming attribute of the Mule message as an ID, or you can compute the ID by configuring a DataWeave expression. You can also use the DataWeave `Crypto` functions to compute hashes(SHA, MD5) from the data.
 
 == Message Hashing
 
-DataWeave allows you to import a `Crypto` library that has some hashing functions defined. This can be useful when you want to compute a unique ID for the message by calculating a hash over it. For example, if you want to get an MD5 hash from the message and use it as the ID, the parameter *'ID Expression* should be:
+DataWeave enables you to import the `Crypto` library to execute hashing functions. Calculating a hash can be useful when you want to compute a unique ID for the Mule message. For example, if you want to get an MD5 hash from the payload and use it as the ID, set the parameter *ID Expression* to the following DataWeave expression:
```

Comment:
Why would a parameter name be in boldface? We use bold for GUI labels, but syntax parameters?